eclass/distutils-r1.eclass

```diff
@@ -750,6 +750,42 @@ distutils_install_for_testing() {
 	esetup.py install "${add_args[@]}" "${@}"
 }
 
+# @FUNCTION: distutils_write_namespace
+# @USAGE: <namespace>...
+# @DESCRIPTION:
+# Write the __init__.py file for the requested namespace into PEP517
+# install tree, in order to fix running tests when legacy namespace
+# packages are installed (dev-python/namespace-*).
+#
+# This function must only be used in python_test().  The created file
+# will automatically be removed upon leaving the test phase.
```

Comment:
Can we mark it as `::gentoo` usage only, so we can remove it when we are ready?
I thought even mark is as deprecated from the start, so we are free to remove it anytime - but this might look weird

Reply:
After some thinking, I'm not 100% sure if we wouldn't need it still. In particular, given that pytest still hasn't implement implicit namespace support.